### ISSUE TITLE
Moving cards

### ISSUE THREAD

**Issue by Trippnology** [websitesfortrello/trelew#13]
Is it possible to move cards from one list to another? Am I missing something obvious?

Thanks for trelew, love it so far!

**Comment by fiatjaf** [comment]
No, it is not implemented. Many corner features like this are missing. I'm implementing them slowly (mostly as I need them). Do you want to contribute with a PR?

(I can help you if you get lost in my super messy code.)

**Comment by Trippnology** [comment]
I'm sorry, I don't have the knowledge to help much.

I guess it could be implemented as `mv` either as a vorpal command (like `ls`) or as a vcommand as part of `enterCard()` (I'm not really sure of the difference).

You would need to know the ids of neighbouring lists to offer them as options, then make a `PUT` request to `/1/cards/[card id]/idList` ([trello docs](https://developers.trello.com/advanced-reference/card#put-1-cards-card-id-or-shortlink-idlist)).

I see the workflow like this:
1. Enter card
2. `mv`
3. Show list of lists on the current board
4. Type name of desired list
5. Optional confirmation.

Moving cards between lists on the same board would be simpler than moving them to another board, which would require an extra step to navigate to the desired target board.

Hope this makes sense!

**Comment by fiatjaf** [comment]
That's great suggestion and interface! `mv` is beautiful. I think it can work for moving the card inside the same list also. `mv 0`, `mv 1`, `mv top`, `mv bottom`, `mv <list>`. I'll try to implement this in the next opportunity. Moving to boards will be left to another opportunity :P

**Comment by Trippnology** [comment]
Yes, moving cards up and down the current list would be useful too, especially with the added keywords of `top` and `bottom`. Good idea!
